news/_posts/core-weekly/2022-07-25-coreweekly-29-2022.md

```diff
@@ -0,0 +1,184 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 29 of 2022"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2022-07-26
+authors: [ PrestaShop ]
+image: /assets/images/2017/04/core_weekly_banner.jpg
+twitter_image: /assets/images/theme/banner-core-weekly.jpg
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 18th to Sunday 24th of July 2022.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+## General messages
+
+Dear developers,
+
+The maintainers team was very busy these last few days because of the [major security vulnerability](https://build.prestashop.com/news/major-security-vulnerability-on-prestashop-websites/) discovered last week. If you have not read this article, please read it now.
```

Comment:
```suggestion
The maintainers team was very busy these last few days because of the [major security vulnerability](/news/major-security-vulnerability-on-prestashop-websites/) discovered last week. If you have not read this article, please read it now.
```
Use relative links